Entities/StudentNumber.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class StudentNumber : BaseEntity
+    {
+        public string Number { get; set; }
+        public List<Guid> Courses { get; set; }
```

Comment:
Is it possibel to find out courses from timetable? If yes why do you need it twice?

Reply:
This is used in side panel. Can I remove it?

Reply:
You must know whether you can. But model should not contain duplicated data... Make method to calculate it if needed... (if it is not too complicated)